src/components/Button.stories.js

```diff
@@ -0,0 +1,10 @@
+import Button from './Button'
+export default {
+  title: 'Button',
+  component: Button,
+}
+
+const Template = args => <Button {...args} />
```

Comment:
Here it would be nice to have also a second version of your button, for example when it is active ;)

Reply:
But I don't think that there are two states of the button (active and inactive) 😉